test/unit/worker.js

```diff
@@ -166,9 +267,61 @@ describe('Worker', function () {
           .then(function () {
             assert.ok(taskHandler.calledOnce, 'task was called once')
             assert.ok(doneHandler.calledOnce, 'done was called once')
+            sinon.assert.calledTwice(monitor.increment)
+            sinon.assert.calledWith(monitor.increment.firstCall, 'ponos', {
+              token0: 'command',
+              token1: 'something.command',
+              token2: 'do.something.command',
+              queue: 'do.something.command'
+            })
+            sinon.assert.calledWith(monitor.increment.secondCall, 'ponos.finish', {
+              result: 'success',
+              token0: 'command',
+              token1: 'something.command',
+              token2: 'do.something.command',
+              queue: 'do.something.command'
+            })
+            sinon.assert.calledOnce(monitor.timer)
+            sinon.assert.calledWith(monitor.timer, 'ponos.timer',
+              !process.env.WORKER_MONITOR_DISABLED, {
+                token0: 'command',
+                token1: 'something.command',
+                token2: 'do.something.command',
+                queue: 'do.something.command'
+              })
+            sinon.assert.calledOnce(timer.stop)
           })
       })
 
+      describe('with disabled monitoring', function () {
+        beforeEach(function () {
+          process.env.WORKER_MONITOR_DISABLED = 'true'
+        })
+
+        afterEach(function () {
+          delete process.env.WORKER_MONITOR_DISABLED
+        })
+
+        it('should run the task and call done and not stop timer', function () {
+          taskHandler = sinon.stub()
+          doneHandler = sinon.stub()
+          return assert.isFulfilled(worker.run())
+            .then(function () {
+              assert.ok(taskHandler.calledOnce, 'task was called once')
+              assert.ok(doneHandler.calledOnce, 'done was called once')
+              sinon.assert.notCalled(monitor.increment)
+              sinon.assert.calledOnce(monitor.timer)
+              sinon.assert.calledWith(monitor.timer, 'ponos.timer',
+                !process.env.WORKER_MONITOR_DISABLED, {
+                  token0: 'command',
+                  token1: 'something.command',
+                  token2: 'do.something.command',
+                  queue: 'do.something.command'
+                })
```

Comment:
either get everything on one line, or put each argument on new lines.

i.e.:

```js
// bad:
assert(foo, bar
  baz, {
    value: 4
  })

// good (the last object started on the same line,
//   which means it closes on the same column as the assert):
assert(foo, bar baz, {
  value: 4
})

// good (new lines all around):
assert(
  foo,
  bar
  baz,
  {
    value: 4
  }
)
```

(sorry to be picky, but I really hate multiple lines with arbitrary numbers of arguments on each line)